src/ape_compile/_cli.py

```diff
@@ -67,7 +68,13 @@ def cli(filepaths, use_cache, display_size):
         else:
             selected_paths = str(project.path / "contracts")
 
-        notify("WARNING", f"No project files detected in '{selected_paths}'")
+        if any(".sol" or ".vy" in s for s in listdir(selected_paths)):
+            if any(".sol" in s for s in listdir(selected_paths)):
+                notify("WARNING", "The required compiler plugin Ape-Solidity is not installed")
+            if any(".vy" in s for s in listdir(selected_paths)):
+                notify("WARNING", "The required compiler plugin Ape-Vyper is not installed")
+        else:
+            notify("WARNING", f"No contract files detected in '{selected_paths}'")
```

Comment:
So, for ape core, we don't take any particular view on which compilers to install, but it is based on extension so we can make that more apparent for the error message
```suggestion
        extensions = ", ".join(set(f.suffix for f in selected_paths))
        notify("WARNING", f"No compilers detected for the following extensions: {extensions}")
```

Reply:
looks good to me :)